ros/api/v0/hosts.py

```diff
@@ -92,8 +92,12 @@ def get(self):
         for row in query_results:
             system_dict = row.System.__dict__
             host = {skey: system_dict[skey] for skey in system_columns}
-            host['recommendation_count'] = len(host['rule_hit_details'])
-            host['state'] = host['rule_hit_details'][0].get('key')
+            if host['rule_hit_details']:
+                host['recommendation_count'] = len(host['rule_hit_details'])
+                host['state'] = host['rule_hit_details'][0].get('key')
+            else:
+                host['recommendation_count'] = 0
```

Comment:
Will it be good to skip such systems which don't have any rule hits?

Reply:
Yes, we can do that.